### PR TITLE
v: Add weekly release channel

### PR DESCRIPTION
**Pull request by issaclin32**
This adds the weekly release channel of [the V language](https://vlang.io/).

### CODE REVIEW

**Review by rashil2000 — APPROVED** [review]

**Comment by niheaven** [comment]
Move it to `Versions`

**Comment by niheaven** [comment]
@issaclin32 @rashil2000 @pratikpc Please move alternative versions to `Versions` buckets and keep `Main` simple and unique.

File: bucket/v-weekly.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "2021.45",
+    "description": "The V programming language (weekly release channel)",
+    "homepage": "https://vlang.io",
+    "license": "MIT",
+    "notes": [
+        "V requires a C compiler on Windows. You can use one of MinGW-w64, llvm-mingw, MSVC.",
```
Comment (pratikpc) [inline bucket/v-weekly.json:7]: V also can download TCC
> V downloads TCC automatically

**Comment by rashil2000** [comment]
> @issaclin32 @rashil2000 @pratikpc Please move alternative versions to `Versions` buckets and keep `Main` simple and unique.

We can do that when we begin the un-deprecation of Nightlies bucket and do a mass migration

**Comment by niheaven** [comment]
> > @issaclin32 @rashil2000 @pratikpc Please move alternative versions to `Versions` buckets and keep `Main` simple and unique.
> 
> We can do that when we begin the un-deprecation of Nightlies bucket and do a mass migration

Yes, many manifests need reorganization.

File: bucket/v-weekly.json (hunk 1 above)
Comment (issaclin32) [reply bucket/v-weekly.json:7]: Thank you for the info. I would be grateful if you can also provide me the source (I assume it's https://github.com/vlang/v#readme).

I think we can remove `V requires a C compiler...` and only keep the description about `Windows SDK`. Is that right?

**Pull request closed without merging**